From cf60428f8c580d2f7b457998f7d031a7746ab57f Mon Sep 17 00:00:00 2001
From: Cherry Chen <cherry.chen@rock-chips.com>
Date: Thu, 23 Aug 2018 17:35:36 +0800
Subject: [PATCH] ARM: dts: rockchip: Put the order of aloop card backwards for
 rk3308 board

For keep other sounds card order, the aloop card index set 7.

Change-Id: Id54369abdd6c4e48679647fe50d84cfe5bcd061b
Signed-off-by: Cherry Chen <cherry.chen@rock-chips.com>
---
 arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts     | 2 +-
 arch/arm/boot/dts/rk3308-dot-v10-aarch32.dts           | 2 +-
 arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts b/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts
index 4aa622f596bb..1640fb90c678 100644
--- a/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts
+++ b/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts
@@ -13,7 +13,7 @@
 	compatible = "rockchip,rk3308-dot-rk816-v10-aarch32", "rockchip,rk3308";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait snd_aloop.index=7";
 	};
 
 	acodec_sound: acodec-sound {
diff --git a/arch/arm/boot/dts/rk3308-dot-v10-aarch32.dts b/arch/arm/boot/dts/rk3308-dot-v10-aarch32.dts
index 813a2d0fded4..02d0c48006b8 100644
--- a/arch/arm/boot/dts/rk3308-dot-v10-aarch32.dts
+++ b/arch/arm/boot/dts/rk3308-dot-v10-aarch32.dts
@@ -13,7 +13,7 @@
 	compatible = "rockchip,rk3308-dot-v10-aarch32", "rockchip,rk3308";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait snd_aloop.index=7";
 	};
 
 	acodec_sound: acodec-sound {
diff --git a/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi b/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
index 78312609f2de..3b075ee653fe 100644
--- a/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
+++ b/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
@@ -11,7 +11,7 @@
 	compatible = "rockchip,rk3308-vmb-v10", "rockchip,rk3308";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait snd_aloop.index=7";
 	};
 
 	adc-keys {
-- 
2.35.3

